server/controllers/Messages/resolver.test.ts

```diff
@@ -0,0 +1,123 @@
+import { expect } from 'chai';
+const faker = require('faker');
+import { callSchema } from '../../testUtils/callSchema';
+
+const emailMutation = `
+  mutation Email($data: SendEmailInputs!) {
+    sendEmail(
+      data: $data
+    ) {
+      ourEmail
+      emailList
+      subject
+      htmlEmail
+    }
+  }
+`;
+
+describe('Test sendEmail resolver', () => {
+  it('Should reject the mutation call when subject field has exceeded MaxLength', async () => {
+    const emailData = {
+      to: ['jonathan.seubert@megajon.com', 'finisher1017@gmail.com'],
```

Comment:
Was having these emails addresses in the code intentional?  Perhaps replace with a .env variable?